src/coreclr/jit/emitarm64.cpp

```diff
@@ -22739,6 +22920,20 @@ void emitter::emitDispSveReg(regNumber reg, insOpts opt, bool addComma)
         emitDispComma();
 }
 
+//------------------------------------------------------------------------
+// emitDispSveRegIndex: Display a scalable vector register with indexed element
+//
+void emitter::emitDispSveRegIndex(regNumber reg, ssize_t index, bool addComma)
+{
+    assert(isVectorRegister(reg));
+    printf("%s", emitSveRegName(reg));
+
+    printf("[%d]", static_cast<int>(index));
```

Comment:
nit: we have `emitDispElementIndex` which will do this.

So something like this:
```cpp
//------------------------------------------------------------------------
// emitDispSveRegIndex: Display a scalable vector register with indexed element
//
void emitter::emitDispSveRegIndex(regNumber reg, ssize_t index, bool addComma)
{
    assert(isVectorRegister(reg));
    printf("%s", emitSveRegName(reg));
    emitDispElementIndex(index, addComma);
}
```